README.md

```diff
@@ -1,93 +1,284 @@
+
+### Steering Council Size
+
+The Steering Council should reflect a wide range of interests, and still be small enough to have a reasonable expectation that [quorum requirements](#quorum) can be met when votes are called.
+
+The minimum and maximum sizes for the Steering Council are:
+
+* Minimum: 9
+* Maximum: 21
+
+The minimum of 9 works well with the [minimum quorum size of 5](#quorum) specified below. The maximum of 21 allows for a diversity of members, including as many as 7 Provisional Members (see below), and representation from 11 to 21 different participating organizations (see below).
+
+If the Steering Council falls below its minimum, then all other Council business is suspended until this requirements is met. The requirement can be met by adding Provisional members, adding new contributing members, or members coming back from Emeritus status.
+
+If the Steering Council membership exceeds its maximum, then all other Council business is suspended until this requirements is met. The requirement can be met by dropping Provisional members, removing members, or members voluntarily moving to Emeritus status.
+
+No vote can be held that would push the Steering Council above its maximum size if the vote resulted in adding either standard or provisional Steering Council members.
+
+### Provisional Members
+
+As mentioned above, Steering Council members will usually have a demonstrated record of prior service to the conda community.  This requirement works well for ensuring that Council members are invested in the community before they can become part of community governance.
+
+We also want to encourage participation by commercial and grant funded organizations that have committed to investing in the conda ecosystem. To encourage these organizations to commit resources to the conda ecosystem, **Provisional Steering Council Memberships** are available.
+
+Provisional memberships are open to organizations that have committed resources to the conda Organization, but do not yet have an established record of contribution *within* the conda Ecosystem. Provisional memberships encourage investment, and give investing organizations an immediate voice in community decisions.
+
+Provisional memberships are open to organizations that
+
+* Are funding at least one FTE contributing to the conda ecosystem
+* Have not been involved in the conda ecosystem previously (and thus have no record of contributing to the ecosystem)
+
+These organizations can request 1 provisional membership on the Steering Council. The evaluation criteria for evaluating this request is based on the organization’s and the nominated person’s commitment to open source principles *as demonstrated in other projects.*
+
+Provisional memberships are expected to be temporary, lasting just long enough for contributors from that organization to demonstrate their values and earn representation as individual contributors.
```

Comment:
From @CJ-Wright

> If this is aimed at generating institutional buy in by organizations then the seat should sit with the organization.

And from @awwad 

> I think CJ is right that this needs some more language to make the provisional seat make sense as something an organization has.

Here I disagree.*  **I think that provisional seats are associated with both companies and individuals.  We want companies involved, but we want their representatives to be potential good community members**, not just whoever the company wants to put on the steering committee.

From @CJ-Wright 

> Should the Council be at maximum capacity that would lead to the disenfranchisement of an organization that otherwise invested in the ecosystem.

I think I am missing something here.  They would only be disenfranchised if the seat were filled while it is open.  Also I think this is a narrow window, as provisional members are only provisional until they can demonstrate that they are good contributors.  

Related to this is a discussion that happened before the vote was called, but that we didn't reach a clear decision on: Is 21 members a good maximum or should we raise that to a higher number?  Raising the maximum makes the disenfranchisement less likely (but still possible).

\* But I don't get a vote!

Reply:
We've talked about this in detail and would suggest to add the following to fix this:

> Organizations may submit an appeal to the Council to stay their provisional membership if and when the organization determines that the individual is no longer acting in the larger interests of the organization or conda community. The Council will consider the appeal and follow the same voting procedure as described in the "[Remove a Provisional Steering Council Member](#remove-a-provisional-steering-council-member)" section below. The organization may then nominate a replacement for the provisional membership following the procedure described in in the “[Nominate a Provisional Steering Council Member](#nominate-a-provisional-steering-council-member)” section below.

Reply:
We've talked about this in detail and would suggest to add the following to fix this:

> Organizations may submit an appeal to the Council to stay their provisional membership if and when the organization determines that the individual is no longer acting in the larger interests of the organization or conda community. The Council will consider the appeal and follow the same voting procedure as described in the "[Remove a Provisional Steering Council Member](#remove-a-provisional-steering-council-member)" section below. The organization may then nominate a replacement for the provisional membership following the procedure described in in the “[Nominate a Provisional Steering Council Member](#nominate-a-provisional-steering-council-member)” section below.

Reply:
SGTM!

Reply:
Weird that both comments made it, I blame the in-flight wifi :)

Reply:
I've made the change in cd9e3442a55c7df64fbd9130bc78dd3c118c1cbd.